pages/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-on-prem/index.md

```diff
@@ -8,7 +8,7 @@ excerpt: A guide for creating Konvoy clusters on your own premises' infrastructu
 
 ## Prerequisites
 
-- Configured [On-Prem Cloud Provider](/ksphere/kommander/latest/operations/cloud-providers/#configuring-an-on-premise-provider)
+- A configured [On-Prem Infrastructure Provider](/ksphere/kommander/latest/operations/cloud-providers/#configuring-an-on-premise-provider)
```

Comment:
This will clash with https://github.com/mesosphere/dcos-docs-site/pull/2991/files#diff-540425a0c55616745fe086cbcc783dbaR11 .